frontend/src/components/offers/offers.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Tab,Tabs } from '@mui/material'; 
+import { useEffect, useState } from "react";
+import productService from '../../services/product.service';
+import Card from '../card/card';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import "./offers.css"
+function Offers() {
+  var tab = 1; 
+  const [products,setProducts] = useState()
+  const count = 8;
+  const [start,setStart] = useState(0)
+  const[more,setMore] = useState(true);
+  function getData(option,start){
+     productService.getNewestOrLastChance(option,start,count).
+        then((response) => { 
+          setStart(start+count);
+          if(response.data.length==0 || response.data.length%count!=0){
+            setMore(false);  
+            console.log(response.data)
```

Comment:
Remove console.logs after creating pull request. If you still need this to be written for some reason, use logger.